doc/buildWithDocker.md

```diff
@@ -66,10 +61,4 @@ The following commands must be run from the root of the project. This operation
 
 ```bash
 docker image build -t infinitime-build ./docker
```

Comment:
Nitpick: I think the `image` in the build command is redundant. I'm always using just `docker build` (could someone with working docker test the command without `image`?)

```suggestion
docker build -t infinitime-build ./docker
```

Reply:
Yep, I tested that command, it runs perfectly!